src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentPageDefault.component.js

```diff
@@ -40,7 +41,7 @@ export const EnrollmentPageDefaultPlain = ({ program, classes }: PlainProps) =>
                 />
             </div>
             <div className={classes.rightColumn}>
-                [placeholder profile widget]
+                <ProfileWidget />
```

Comment:
There was this thread in the PR where we introduced the Widgets folder where we discussed that we wanted to have each widget to live in its own inside the `/components` folder directly. Simona, suggested we go with the naming `Widget.....`. 

I am wondering if we want to start applying this from this widget onwards, what you think?

Reply:
Absolutely. I will change it